napari/_tests/test_viewer.py

```diff
@@ -33,15 +41,12 @@ def test_viewer(make_napari_viewer):
     assert viewer.dims.ndisplay == 3
     viewer.dims.ndisplay = 2
     assert viewer.dims.ndisplay == 2
-
     # Run all class key bindings
-    for func in viewer.class_keymap.values():
-        # skip fullscreen test locally
-        if func.__name__ == 'toggle_fullscreen' and not os.getenv("CI"):
-            continue
-        if func.__name__ == 'play':
-            continue
-        func(viewer)
+    if func.__name__ == 'toggle_fullscreen' and not os.getenv("CI"):
```

Comment:
Can you pull this out into its own test, `tes_viewer_bindings`, that doesn't have any of the `assert` stuff above, no need for that to get tested each time